extensions/positron-r/src/provider.ts

```diff
@@ -293,58 +293,60 @@ function binFragments(): string[] {
 }
 
 export async function findCurrentRBinary(): Promise<string | undefined> {
-	if (os.platform() === 'win32') {
-		const registryBinary = findCurrentRBinaryFromRegistry();
-		if (registryBinary) {
-			return registryBinary;
-		}
-	}
+	// if (os.platform() === 'win32') {
+	// 	const registryBinary = findCurrentRBinaryFromRegistry();
+	// 	if (registryBinary) {
+	// 		return registryBinary;
+	// 	}
+	// }
 
 	const whichR = await which('R', { nothrow: true }) as string;
 	if (whichR) {
+		LOGGER.info(`Found R on PATH: ${whichR}.`);
 		if (os.platform() === 'win32') {
-			// rig puts {R Headquarters}/bin on the PATH, so that is probably how we got here.
-			// (The CRAN installer does NOT put R on the PATH.)
-			// In the rig scenario, `whichR` is anticipated to be a batch file that launches the
-			// current version of R ('default' in rig-speak):
-			// Example filepath: C:\Program Files\R\bin\R.bat
-			// Typical contents of this file:
-			// ::4.3.2
-			// @"C:\Program Files\R\R-4.3.2\bin\R" %*
-			// How it looks when r-devel is current:
-			// ::devel
-			// @"C:\Program Files\R\R-devel\bin\R" %*
-			// Note that this is not our preferred x64 binary, so we try to convert it.
-			if (path.extname(whichR).toLowerCase() === '.bat') {
-				const batLines = readLines(whichR);
-				const re = new RegExp(`^@"(.+R-(devel|[0-9]+[.][0-9]+[.][0-9]+).+)" %[*]$`);
-				const match = batLines.find((x: string) => re.test(x))?.match(re);
-				if (match) {
-					const whichRMatched = match[1];
-					const whichRHome = getRHomePath(whichRMatched);
-					if (!whichRHome) {
-						LOGGER.info(`Failed to get R home path from ${whichRMatched}`);
-						return undefined;
-					}
-					// we prefer the x64 binary
-					const whichRResolved = firstExisting(whichRHome, binFragments());
-					if (whichRResolved) {
-						LOGGER.info(`Resolved R binary at ${whichRResolved}`);
-						return whichRResolved;
-					} else {
-						LOGGER.info(`Can\'t find R binary within ${whichRHome}`);
-						return undefined;
-					}
-				}
-			}
-			// TODO: handle the case where whichR isn't picking up the rig case; do people do this,
-			// meaning put R on the PATH themselves, on Windows?
+			return await findCurrentRBinaryFromPATHWindows(whichR);
 		} else {
-			const whichRCanonical = fs.realpathSync(whichR);
-			LOGGER.info(`Resolved R binary at ${whichRCanonical}`);
-			return whichRCanonical;
+			return await findCurrentRBinaryFromPATHNotWindows(whichR);
 		}
 	}
+	return undefined;
+}
+
+async function findCurrentRBinaryFromPATHWindows(whichR: string): Promise<string | undefined> {
+	// The CRAN Windows installer does NOT put R on the PATH.
```

Comment:
> The R installer does not put Rcmd.exe (nor any other R executable) on your PATH. 

https://cran.r-project.org/bin/windows/base/rw-FAQ.html#Rcmd-is-not-found-in-my-PATH_0021-1